CRM/Member/BAO/Membership.php

```diff
@@ -2398,6 +2398,8 @@ public static function recordMembershipContribution(&$params, $ids = array()) {
       'contribution_recur_id',
       'pan_truncation',
       'card_type_id',
+      'partial_payment_total',
+      'partial_amount_pay',
```

Comment:
Just wondering about name: should it be partial_amount_paid if it represents what is being paid now? If it represents what remains to be paid, how about partial_amount_to_pay ?